packages/compass-settings/src/components/modal.tsx

```diff
@@ -54,7 +55,10 @@ const footerStyles = css({
   paddingBottom: 0,
 });
 
-const settings: Settings[] = [{ name: 'Privacy', component: PrivacySettings }];
+const settings: Settings[] = [
+  { name: 'Privacy', component: PrivacySettings },
+  { name: 'Theme', component: ThemeSettings },
```

Comment:
Oh, one small thing here. In the menu we mention that this is still a "Preview" feature, should we add it to the label here too?

![image](https://user-images.githubusercontent.com/5036933/198239562-ec629b21-7096-455f-b498-3241b5adf560.png)

Reply:
Yes! :+1:

Reply:
Looks good, thanks for updating!